### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: temba/base.py [hunk 1]
```diff
@@ -305,8 +310,7 @@ def _request(self, method, url, body=None, params=None):
                    'Authorization': 'Token %s' % self.token,
                    'User-Agent': user_agent_header}
 
-        if self.debug:  # pragma: no cover
```
Comment (rowanseymour) [inline temba/base.py:308]: Can remove this param from `__init__` as well since no longer needed

**Comment by rowanseymour** [comment]
With removal of that `debug` param this looks good. Really appreciate this effort to get all this code more up to date.

**Comment by ghost** [comment]
Updated.

File: temba/__init__.py [hunk 2]
```diff
@@ -13,7 +13,6 @@ class TembaClient(AbstractTembaClient):
     :param str host: server hostname, e.g. 'rapidpro.io'
     :param str token: organization API token
     :param str user_agent: string to be included in the User-Agent header
-    :param bool debug: print debugging statements
     """
     def __init__(self, host, token, user_agent=None, debug=False):
```
Comment (rowanseymour) [inline temba/__init__.py]: `debug` param still here tho removed on base class

**Comment by ghost** [comment]
Luckily the automatic test coverage caught my mistakes. It's updated & tests pass.

**Pull request merged**